lib/browser/api/extensions.js

```diff
@@ -538,7 +538,13 @@ ipcMain.on('chrome-cookies-getall', function (evt, responseId, details) {
 ipcMain.on('chrome-cookies-get', function (evt, responseId, details) {
   const cb = (error, cookie) => {
     if (!evt.sender.isDestroyed()) {
-      evt.sender.send('chrome-cookies-get-response-' + responseId, error, cookie)
+      var first = cookie
```

Comment:
@bridiver An array of cookies is returned from `getAll`, whereas a single cookie is returned from `get`. To your second point, our implementation is shallow indeed. We'll definitely need to agument to add support for the finer filtering.